rescript

```diff
@@ -51,6 +69,20 @@ function updateFinishTime() {
   var diff = process.hrtime(startTime);
   return diff[0] * 1e9 + diff[1];
 }
+
+/**
+ *
+ * @param {string} file
+ * @returns
+ */
+function getWatchFiles(file) {
+  if (fs.existsSync(file)) {
+    return JSON.parse(fs.readFileSync(file, "utf8"));
```

Comment:
How do you know the file still exists here?

Reply:
This piece of code change is not relevant to this diff, it is just shuffling around.
Everytime when build finished, we do update the watched directories